yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
Hahah I've done this before too - I'm guessing you ran yarn before cd-ing into support-frontend?

Reply:
(we can delete this)

Reply:
Yeah, I must've done!

Reply:
I've removed this now, thanks for the spot @TomPretty!